docs/rules/catch-error-name.md

```diff
@@ -75,8 +75,40 @@ somePromise.catch(_ => {
 
 ## Options
 
+### name
+
 You can set the `name` option like this:
 
 ```js
 "unicorn/catch-error-name": ["error", {"name": "err"}]
 ```
+
+### caughtErrorsIgnorePattern
+
+```js
+"unicorn/catch-error-name": ["error", {"caughtErrorsIgnorePattern": "^ignore"}]
+```
+
+The `caughtErrorsIgnorePattern` option specifies exceptions not to check for usage: catch arguments whose names match a regexp pattern.
```

Comment:
This sentence could use some more work. It doesn't read very well.

Reply:
@sindresorhus also was got from https://eslint.org/docs/rules/no-unused-vars#caughterrorsignorepattern. It is difficult to imagine how to explain in more detail